sdk/core/azure-core/src/main/java/com/azure/core/util/polling/strategy/PollingStrategy.java

```diff
@@ -0,0 +1,111 @@
+     */
+    Mono<LongRunningOperationStatus> onActivationResponse(Response<?> response, PollingContext<BinaryData> context);
+
+    /**
+     * Gets the URL to poll the long running operation status from the current polling context.
+     *
+     * @param context the {@link PollingContext} for the current polling operation
+     * @return the URL to send a GET request to get the latest long running operation status
+     */
+    String getPollingUrl(PollingContext<BinaryData> context);
+
+    /**
+     * Parses the response from the polling URL into a {@link LongRunningOperationStatus}, and stores information
+     * useful for further polling and final response in the {@link PollingContext}.
+     *
+     * @param response the response from the polling URL from the past poll
+     * @param context the {@link PollingContext} for the current polling operation
+     * @return a publisher emitting the long running operation status
+     */
+    Mono<LongRunningOperationStatus> onPollingResponse(HttpResponse response, PollingContext<BinaryData> context);
+
+    /**
+     * Gets the URL to get the final result of the long running operation from the current polling context. Returns
+     * null if no final GET call is needed.
+     *
+     * @param context the {@link PollingContext} for the current polling operation
+     * @return the URL to send a GET request to get the final result
+     */
+    String getFinalGetUrl(PollingContext<BinaryData> context);
+
+    /**
+     * Parses the response from the final GET call into the result type of the long running operation.
+     *
+     * @param response the response from the final GET URL
+     * @param context the {@link PollingContext} for the current polling operation
+     * @param resultType the {@link Type} of the final result object to deserialize into, or BinaryData if raw response
+     *                   body should be kept. This should match the generic parameter {@link U}.
+     * @param <U> The type of the final result of long running operation
+     * @return a publisher emitting the final result
+     */
+    <U> Mono<U> getFinalResult(HttpResponse response, PollingContext<BinaryData> context, Type resultType);
+
+    /**
+     * Creates a default polling strategy based on the {@link ChainedPollingStrategy} implementation, trying in the
+     * order of {@link OperationResourcePollingStrategy}, {@link LocationPollingStrategy}, and {@link StatusCheckPollingStrategy},
+     * and polls with the first strategy that can poll the current long running operation.
+     *
+     * @return an instance of the default polling strategy
+     */
+    static PollingStrategy createDefault() {
+        return new ChainedPollingStrategy()
+            .addPollingStrategy(new OperationResourcePollingStrategy())
+            .addPollingStrategy(new LocationPollingStrategy())
+            .addPollingStrategy(new StatusCheckPollingStrategy());
+    }
```

Comment:
Do we expect LLC to use this `PollingStrategy.createDefault` for LRO client method?

Reply:
It's moved to `ChainedPollingStrategy`. Sample usage:

```java
    public PollerFlux<BinaryData, BinaryData> beginHealthAsync(MultiLanguageBatchInput input, String modelVersion, StringIndexType stringIndexType) {
        return PollerFlux.create(Duration.ofSeconds(1),
            () -> this.healthWithResponseAsync(input, modelVersion, stringIndexType),
            com.azure.core.util.polling.ChainedPollingStrategy.createDefault(this.getHttpPipeline(), Context.NONE),
            new TypeReference<>() { }, new TypeReference<>() { });
    }
```